spec/lib/binary-buffer-module.spec.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+describe("binaryBuffer", () => {
+    var binaryBuffer, copy;
+
+    beforeEach(() => {
+        binaryBuffer = require("../../lib/binary-buffer")();
+    });
+    [
+        "some string",
+        "",
+        new Buffer("some buffer"),
+        123
```

Comment:
Tests should cover all situations that you're expecting and a few that even should not happen.  Tests are a way of defining your routine's behavior.  So, maybe at times throw odd stuff at a routine like you did here - that's why I didn't say it was a bad idea.  I just think that the tests are broken because they are passing when 123 is used.